conjur/argument_parser/_init_parser.py

```diff
@@ -0,0 +1,48 @@
+import argparse
+from conjur.argument_parser.parser_utils import command_description, command_epilog, formatter, title_formatter
+
+
+class InitParser:
+
```

Comment:
docstring?

Reply:
Added in all files + constructor that raises error if tried to initialize the class not as part of the argparse builder